nano/node/node.cpp

```diff
@@ -1622,6 +1622,7 @@ node (init_a, io_ctx_a, application_path_a, alarm_a, nano::node_config (peering_
 }
 
 nano::node::node (nano::node_init & init_a, boost::asio::io_context & io_ctx_a, boost::filesystem::path const & application_path_a, nano::alarm & alarm_a, nano::node_config const & config_a, nano::work_pool & work_a) :
+startup_time (std::chrono::system_clock::now ()),
```

Comment:
Effective C++ (Scott Meyers) - Item 13 

> "List members in an initialization list in the order in which they are declared"
> Reason for rule: Class members are initialized in the order of their declaration in the class, not by the order in which they are listed in a member initialization list.

If you want `startup_time` to be called before other members then it should be listed at the top in the class definition.

Reply:
Done, moved it to the bottom of the initialization order.